src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__description {
+    color: $main-text-color;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+  }
+
+  &__code {
+    color: $secundary-text-color;
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-reviews {
+      color: $main-text-color;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+
+    &-text {
+      color: $secundary-text-color;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
```

Comment:
The line-height of the price value is set to 18px. This is the same as the font-size, which might make the text look cramped. Consider increasing the line-height to improve readability and the overall look of the text.